intf/cnsr_intf.lua

```diff
@@ -98,28 +102,28 @@ function display_reason(reason_string, category, tag_end)
 end
 
 function check_disable_actions()
-	if mute_params.activated and (current_time > mute_params.end_time or current_time < mute_params.start_time) then
-		vlc.volume.set(mute_params.prev_volume)
-		mute_params.activated = false
+	if MuteParams.activated and (current_time > MuteParams.end_time or current_time < MuteParams.start_time) then
+		vlc.volume.set(MuteParams.prev_volume)
+		MuteParams.activated = false
 	end
 
-	if hide_params.activated and (current_time > hide_params.end_time or current_time < hide_params.start_time) then
+	if HideParams.activated and (current_time > HideParams.end_time or current_time < HideParams.start_time) then
 		vlc.var.set(hide_filter, "video-filter", "")
-		hide_params.activated = false
+		HideParams.activated = false
 	end
 end
 
 function skip(skip_start, skip_end)
```

Comment:
should receive input because it is no longer global

Reply:
which input? I'm not sure I understand which var you refer to

Reply:
skip uses input = vlc.object.input()

Reply:
right, but `check_disable_actions` doesn't use `skip` function

Reply:
I'm referring to the skip function signature line 116

Reply:
oh for some reason I see that you refer to line 112